global-setup.ts

```
@@ -1,16 +1,21 @@
+// Import default logger as fallback
+import { closeAllLoggers, defaultLogger } from './src/log/logger';
+import { resetTestItemsFile } from './src/utils/testItemExporter';
 import { FullConfig } from '@playwright/test';
-import { Logger } from 'pino'; // Import Logger type
-import { closeAllLoggers, defaultLogger } from './src/log/logger'; // Import default logger as fallback
+import { Logger } from 'pino';
 
 async function globalSetup(config: FullConfig) {
```

ESLint

[error] 'config' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)